azurerm/internal/services/compute/managed_disk_resource_test.go

```diff
@@ -322,6 +322,117 @@ func TestAccManagedDisk_attachedStorageTypeUpdate(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMManagedDisk_create_withNetworkPolicy(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_managed_disk", "test")
+	r := ManagedDiskResource{}
+
+	data.ResourceTest(t, r, []resource.TestStep{
+		{
+			Config: testAccAzureRMManagedDisk_create_withNetworkPolicy(data),
+			Check: resource.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
```

Comment:
after every step in every test we generally have an import step, could we add these please?